src/common/iop_profile.c

```diff
@@ -857,7 +857,7 @@ dt_iop_order_iccprofile_info_t *dt_ioppr_get_histogram_profile_info(struct dt_de
   const char *histogram_profile_filename;
   dt_ioppr_get_histogram_profile_type(&histogram_profile_type, &histogram_profile_filename);
   return dt_ioppr_add_profile_info_to_list(dev, histogram_profile_type, histogram_profile_filename,
-                                           DT_INTENT_PERCEPTUAL);
+                                           INTENT_RELATIVE_COLORIMETRIC);
```

Comment:
Should this be `DT_INTENT_RELATIVE_COLORIMETRIC`?

Reply:
This one is a call into dt code so should use DT_* constant.

Reply:
Let's change it to use the `DT_` prefix.